cruft/_cli.py

```diff
@@ -231,3 +231,19 @@ def update(
         strict=strict,
     ):
         raise typer.Exit(1)
+
+
+@app.command(
+    short_help="Show the diff between the project and the current cruft template.",
+    help=_get_help_string(_commands.diff),
+)
+def diff(
+    project_dir: Path = typer.Option(
```

Comment:
Can we replicate all of the flags from cruft check here?

Reply:
I'm not sure if this was a question or a request. `cruft check` command has 3 options:
- `--project-dir` that is already replicated in `cruft diff`
- `--checkout` that allows to specify against which branch to update
- `--strict` that I don't understand what it does but seems tightly related to the update process

So I'm not sure what I else could replicate here. In the meantime, I kind of like that `cruft diff` just diffs the local template against the remote one, following what's in the current `.cruft.json` and regardless of any update process. It just makes it more predictable and easy to reason about IMHO.

Tell me if you disagree and what you think I should replicate.

Reply:
I think checkout would definitely be useful here.

Reply:
I see. Should be quite easy to implement.